pkg/kubelet/kubelet_pods.go

```diff
@@ -1173,6 +1173,15 @@ func (kl *Kubelet) HandlePodCleanups(ctx context.Context) error {
 	metrics.RestartedPodTotal.WithLabelValues("true").Add(float64(restartCountStatic))
 	metrics.RestartedPodTotal.WithLabelValues("").Add(float64(restartCount))
 
+	// Complete termination of deleted pods that are not runtime pods (don't have
+	// running containers), are terminal, but are not known to pod workers.
+	// The example is rejected pods. Triggering TerminatePod allows to proceed
```

Comment:
An example is pods rejected during kubelet admission that have never started before (i.e. do not have an orphaned pod).

Reply:
Used the text :)